src/parameter.jl

```diff
@@ -61,6 +61,13 @@ struct Parameter{T, F, InPlace}
 end
 Parameter(f, model) = Parameter{typeof(f())}(f, model)
 
+"""
+    Parameter(a::AbstractArray, model)
+
+Create a new 'in-place' `Parameter` from `a`, whose values will be used to solve `model`.
+"""
+Parameter(a::AbstractArray, model) = Parameter(()->a, model)
```

Comment:
See also https://github.com/tkoolen/Parametron.jl/issues/67 (which needs resolution). I don't think it's a good idea to make `AbstractArray` special in this sense, and the `Parameter{T}(f::F, model) where {T, F}` constructor already takes over the general case. What do you think about the options discussed in #67?

Reply:
I'll comment there.